addons/stock_landed_costs/models/account_move_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
I think this file will not be accepted for the licence. Can you change it accordingly?

Reply:
Hi @nhomar !
Thanks for your review.
I changed the file according to your comment.